action.yml

```diff
@@ -14,6 +14,8 @@ inputs:
 runs:
   using: composite
   steps:
-    - name: Get Token
+    - run: echo "${{ github.action_path }}" >> $GITHUB_PATH
```

Comment:
Do we need to "reset" this to avoid polluting the system?

Alternatively, would it be possible to run `bash ${{ github.action_action }}/get-token.sh`?

Reply:
> Do we need to "reset" this to avoid polluting the system?

I am not sure about this, but your suggestion is a good idea.